repository/src/main/java/ru/karalionak/rest/mapper/ParametersMapper.java

```diff
@@ -0,0 +1,7 @@
+package ru.karalionak.rest.mapper;
+
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+
+public interface ParametersMapper<ENTITY> {
```

Comment:
You can replace mappers with out of box solution. Please, check BeanPropertySqlParameterSource.